hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/execution/flowcontrol/ExecutionContext.java

```diff
@@ -27,10 +27,15 @@ public final class ExecutionContext {
   private final TermIndex termIndex;
 
   private ExecutionContext(long index, TermIndex termIndex) {
-    this.index = index;
     if (null == termIndex) {
       termIndex = TermIndex.valueOf(-1, index);
     }
+    if (index == -1) {
```

Comment:
Just curious, when ternIndex will be a valid null?

Reply:
termIndex will be null in new flow, pre-execute case which is before pre-ratis transaction at leader node.

So to avoid any null pointer case, precaution check is added. This will be cleaned-up after movement to new flow